doc/nrf/gs_programming.rst

```diff
@@ -29,28 +29,29 @@ You can download it from the following links:
 
 1. Extract the downloaded package and run the file :file:`bin/emStudio`.
 
-#. Select **File -> Open nRF Connect SDK Project.**
+#. Select **File -> Open nRF Connect SDK Project**.
 
-    .. figure:: images/ses_open.JPG
+    .. figure:: images/ses_open.png
        :alt: Open nRF Connect SDK Project menu
 
        Open nRF Connect SDK Project menu
 
-#. To import a project into SES, you must specify the following paths:
+#. The first time you import an |NCS| project, SES will prompt you to set the paths to the Zephyr Base directory and the GNU ARM Embedded Toolchain.
```

Comment:
Should we provide a screen dump here as well ?

Reply:
added